src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -130,13 +140,21 @@ abstract class AbstractCheckTask(
     private var bufferContainsStartMessage: Boolean = false
     private var isDefaultSequence: Boolean = false
 
-    override fun onStart() {
+    @Volatile
+    protected var started = false
```

Comment:
```suggestion
    protected var started = false
      private set
```

Also, how does it solve the problem with LazyInitException? Each rule needs to check this property?